src/applications/disability-benefits/all-claims/Form526EZApp.jsx

```diff
@@ -270,4 +273,21 @@ const mapStateToProps = state => ({
   user: state.user,
 });
 
+if (isLoggedIn) {
```

Comment:
I'd suggest adding a condition to make sure this doesn't run during CI, see [here](https://github.com/department-of-veterans-affairs/vets-website/blob/025469ecdf03146f5da04355523ee22e02acd205/src/applications/check-in/hooks/useDatadogRum.jsx#L11C5-L11C53) for an example...